scripts/python/phoedf.py

```diff
@@ -75,7 +75,7 @@ def flatten_indices(mu, nu):
     self.gcov = np.zeros([self.NumBlocks, 4, 4, self.Nx3, self.Nx2, self.Nx1])
     for mu in range(4):
       for nu in range(4):
-        self.gcov[:,mu,nu,:,:,:] = self.flatgcov[:,flatten_indices(mu,nu),:,:,:]
```

Comment:
I'm not immediately sure why this change was necessary, I should figure this out

Reply:
It was due to lack of ghost zone pruning for non-proper-variable data like `gcov` after the recent changes to parthenon output. This is properly fixed in #165